pkgs/os-specific/linux/amdgpu-pro/default.nix

```diff
@@ -1,167 +1,206 @@
-{ lib, stdenv, fetchurl, elfutils
-, xorg, patchelf, openssl, libdrm, udev
-, libxcb, libxshmfence, libepoxy, perl, zlib
-, ncurses
-, libsOnly ? false, kernel ? null
```

Comment:
Is it not possible to use just the libs anymore? Using just the AMF libs for HW accelerated video encoding with an otherwise fully AMDGPU (non-pro) system would be a use-case here for example.